content/300-guides/200-deployment/110-deployment-guides/550-deploying-to-deno-deploy.mdx

```diff
@@ -0,0 +1,229 @@
+---
+title: 'Deploying to Deno Deploy'
+metaTitle: 'Deploying to Deno Deploy'
+metaDescription: 'Learn how to deploy a TypeScript application to Deno Deploy.'
+---
+
+<TopBlock>
+
+Today you'll be building and deploying a simple function to Deno Deploy that uses Prisma to save every request to a PostgreSQL database for inspection later.
+
+This guide covers Prisma Client, Data Proxy, and Deno Deploy. Let's get started!
+
+</TopBlock>
+
+## Prerequisites
+
+- Free [Deno Deploy](https://deno.com/deploy) account
+- Free [GitHub](https://github.com/) account
+- A PostgreSQL database
+- Node.js & NPM installed
+- Git installed
+
+## 1. Set up your Application
+
+Open your terminal and navigate to a location of your choice. Run the following commands to set up your application.
+
+```terminal
+mkdir prisma-deno-deploy
+cd prisma-deno-deploy
+deno run -A --unstable npm:prisma init
+```
+
+Inside the generated `prisma/schema.prisma`, add the following schema:
+
+```prisma
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Log {
+  id      String @id @default(autoincrement())
+  level   Level
+  message String
+  meta    Json
+}
+
+enum Level {
+  Info
+  Warn
+  Error
+}
+```
+
+This data model will be used to store incoming requests from your application.
+
+In your `.env` file, replace the current placeholder connection string with your PostgreSQL connection string.
+
+## 2. Create the database schema
+
+With the data model created and your database connection configured, you can now create the data model in your database:
+
+```terminal
+deno run -A --unstable npm:prisma db push
+```
+TODO does this already generate?
```

Comment:
Yes it does

Reply:
But not with the data proxy

Reply:
which puts the query engine in the client folder which then later does not get removed... *sigh

Reply:
Let's merge this PR soon https://github.com/prisma/prisma/pull/13702